wikimedia/wikipedia-ios
# Only reload all widget timelines from the main app

### Notes
To avoid a situation where updating the explore feed data from the widget app extension would cause an infinite update loop, only reload widget timelines from the main app

### Test Steps
1. Open the widget when it needs to reload data from the network
2. Verify that `WidgetCenter.shared.reloadAllTimelines()` isn't called
